test/runnable/extra-files/externmangle.cpp

```diff
@@ -215,6 +215,13 @@ size_t testCppSizeTMangle(ptrdiff_t a, size_t b)
     return a + b;
 }
 
+void testCppI8Mangle (int8_t,  uint8_t,  int_least8_t,  uint_least8_t,  int_fast8_t,  uint_fast8_t)  {}
+void testCppI16Mangle(int16_t, uint16_t, int_least16_t, uint_least16_t, int_fast16_t, uint_fast16_t) {}
+void testCppI32Mangle(int32_t, uint32_t, int_least32_t, uint_least32_t, int_fast32_t, uint_fast32_t) {}
+void testCppI64Mangle(int64_t, uint64_t, int_least64_t, uint_least64_t, int_fast64_t, uint_fast64_t) {}
+void testCppIntPtrMangle(intptr_t, uintptr_t) {}
+void testCppIntMaxMangle(intmax_t, uintmax_t) {}
```

Comment:
It would be nice to make them return a unique value, so that we can be sure the right function is actually called.

Reply:
I can live with that.